packages/netlify-cms-backend-gitlab/src/API.ts

```diff
@@ -41,6 +41,7 @@ export interface Config {
   repo?: string;
   squashMerges: boolean;
   initialWorkflowStatus: string;
+  includeCookiesInRequests?: boolean;
```

Comment:
Same as the other GitHub related comment, this code in this file is applied when using the GitLab backend directly (without git-gateway).
I don't think we would like to have the new logic when not using git-gateway

Reply:
Same as above, this feature is only active when using git gateway.